bip-0126.mediawiki

```diff
@@ -0,0 +1,98 @@
+<pre>
+  BIP: 126
+  Title: Best Practices for Heterogeneous Input Script Transactions
+  Author: Kristov Atlas <kristov@openbitcoinprivacyproject.org>
```

Comment:
I transcribed the results of the discussions that occurred during the OBPP weekly meetings.